core/src/test/scala/cats/derived/empty.scala

```diff
@@ -52,32 +52,37 @@ class EmptySuite extends KittensSuite {
   {
     import auto.empty._
     testEmpty("auto")
-    illTyped("Empty[IList[Int]]")
-    illTyped("Empty[Snoc[Int]]")
+    // NOTE: These typecheck but crash the compiler on Scala 2.13
+//    println(Empty[IList[Int]])
+//    println(Empty[Snoc[Int]])
```

Comment:
cc @milessabin I don't think I will be able to minimize this enough to open a Scala issue.

Reply:
Can you post the stacktrace as a gist?

Reply:
Sure: https://gist.github.com/joroKr21/a62574d04b734628f52c0c6d4a89c7d3
There are two files: `jvm.txt` and `js.txt`, because the traces are different.